app/models/taxon.rb

```diff
@@ -68,4 +74,11 @@ def can_subscribe?
 
     true
   end
+
+private
+
+  def fetch_tagged_content
+    taxon_content_ids = [content_id] + associated_taxons.map(&:content_id)
+    TaggedContent.fetch(*taxon_content_ids)
```

Comment:
Yep. I'm wondering whether the whole splat thing is a bit messy. In the test helper I've just made it a string or an array. https://github.com/alphagov/collections/pull/348/commits/77c6592ecf9ccbeebc2e5e2dcf4dab1d62def2a3#diff-c3f5864092d1f31b743d969d943bd38cR2 Is that clearer?